samples/bluetooth/nrf_dm/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -33,6 +33,11 @@
 			mbox-names = "tx", "rx";
 			role = "host";
 			status = "okay";
+
+			bt_hci_ipc0: bt_hci_ipc0 {
```

Comment:
Should this overlay also be added for the Audio DK?

Reply:
That should be needed as the audio board reuses existing configurations. This sample deletes the IPC node which requires us to put it back.


The file `boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp.dts` includes `<nordic/nrf5340_cpuapp_qkaa.dtsi>` which includes `#include <nordic/nrf5340_cpuapp.dtsi>`  which again includes `dts/arm/nordic/nrf5340_cpuapp_ipc.dtsi` which defined the HCI IPC interface